RedditSharp/Reddit.cs

```diff
@@ -358,29 +378,34 @@ public Thing GetThingByFullname(string fullname)
             var data = WebAgent.GetResponseString(response.GetResponseStream());
             var json = JToken.Parse(data);
             return Thing.Parse(this, json["data"]["children"][0], WebAgent);
+        }
+
+        public Comment GetComment(string subreddit, string name, string linkName)
+        {
+            try
+            {
+                if (linkName.StartsWith("t3_"))
+                    linkName = linkName.Substring(3);
+                if (name.StartsWith("t1_"))
+                    name = name.Substring(3);
+
+                var url = string.Format(GetCommentUrl, subreddit, linkName, name);
+                var fullUrl = string.Format(GetPostUrl, url);

```

Comment:
Not sure this should be changed, and it's formatting GetPostUrl to get a full url for a comment which doesn't seem right.